docs/src/views/FAQ.tsx

```diff
@@ -28,22 +28,13 @@ export default () => (
     <Answer
       content={
         <p>
-          We recommend to render links from react router as Buttons as below:
+          We recommend to render links from React Router as Buttons as below:
           <CodeSnippet
             value={`
-      <>
-          import { Link } from 'react-router-dom'
-          
-          <Button
-            as={Link}
-            content={previous.name}
-            icon="arrow left"
-            iconPosition="before"
-            primary
-            to={previous.url}
-          />
-      </>
-      `}
+            import { Link } from 'react-router-dom'
```

Comment:
Updated to be more clear...

### Before

![image](https://user-images.githubusercontent.com/14183168/57519790-6dcde780-731c-11e9-8b53-c067fd8fa0d7.png)


### After 

![image](https://user-images.githubusercontent.com/14183168/57519587-eed8af00-731b-11e9-825e-62a6631cbf57.png)